lfads/lfads.py

```diff
@@ -769,8 +768,6 @@ def encode_data(dataset_bxtxd, enc_cell, name, forward_or_reverse,
     # Variational Lower Bound on posterior, p(z|x), plus reconstruction cost.
     # KL and reconstruction costs are normalized only by batch size, not by
     # dimension, or by time steps.
-    kl_cost_g0_b = tf.zeros_like(batch_size, dtype=tf.float32)
-    kl_cost_co_b = tf.zeros_like(batch_size, dtype=tf.float32)
```

Comment:
@jazcollins the deletion was just to maintain the current behaviour - updating as per your suggestion but this would suggest that part of the code isn't tested at the moment (or if it is the test needs to be updated)